### ISSUE TITLE
"npm init video" fails with "No matching version found"

### ISSUE THREAD

**Issue by ismaail** [remotion-dev/remotion#1887]
`npm init video` gives the following error `No matching version found for @remotion/cli@3.3.60-alpha.0`

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @remotion/cli@3.3.60-alpha.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2023-03-02T18_00_38_292Z-debug-0.log
Error: Command failed with exit code 1: npm install --no-fund
    at makeError (~/.npm/_npx/40176cad2f2d1409/node_modules/execa/lib/error.js:60:11)
    at handlePromise (~/.npm/_npx/40176cad2f2d1409/node_modules/execa/index.js:118:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async init (~/.npm/_npx/40176cad2f2d1409/node_modules/create-video/dist/init.js:150:9) {
  shortMessage: 'Command failed with exit code 1: npm install --no-fund',
  command: 'npm install --no-fund',
  escapedCommand: 'npm install --no-fund',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
npm ERR! code 1
npm ERR! path ~/Workspace/Projects/Work/Quiz/remotion
npm ERR! command failed
npm ERR! command sh -c -- create-video

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2023-03-02T18_00_34_263Z-debug-0.log
```

- node: v19.7.0
- npm: 8.19.2
- OS: archlinux

**Comment by ismaail** [comment]
workaround

run `yarn create video`

installation will starts,
then when it shows the error: `Couldn't find any versions for "@remotion/cli" that matches "3.3.60-alpha.0"`
you'll get the question `? Please choose a version of "@remotion/cli" from this list`

scroll  down to the choice `3.3.59` and continue the installation

`yarn start` launchs the app in the browser successfully.

**Comment by JonnyBurger** [comment]
Sorry about that! I tried a prerelease today but it got tagged as @latest and nom wouldn't let me retract it :/ 

I've now released v3.3.60 which means the issue should be fixed!
